testUtils/pgMocker.js

```diff
@@ -0,0 +1,62 @@
+const DUMMY_ROW = { name: 'LumigoDevMan', dummyNumber: 6 };
+
+export const createMockedResponse = (rowCount = 5) => {
+  const rows = [];
+  for (let i = 0; i < rowCount; i++) {
+    rows.push({ ...DUMMY_ROW });
+  }
+  return {
+    command: 'SELECT',
+    rowCount: rowCount,
+    oid: null,
+    rows: rows,
+    fields: [],
+    _parsers: [],
+    _types: [],
+    RowCtor: null,
+    rowAsArray: false,
+  };
+};
+const createMockClientWithOptions = mockedOptions => {
+  const { error, rowCount } = mockedOptions;
+  const Client = function Client(options = {}) {
```

Comment:
```suggestion
  const Client = function(options = {}) {
```
Don't we omit the name of the function if we do `const func = ...`? 🤔

Reply:
You right, its was by mistake (writen and moved to const)